fail2ban/client/configreader.py

```diff
@@ -285,8 +285,10 @@ def getOptions(self, pOpts):
 		
 		if self.has_section("Init"):
 			for opt in self.options("Init"):
+				v = self.get("Init", opt)
```

Comment:
`v` will be use in 2 lines below. Please let such easy code fixing not as a hand brake for such PR's again. I mean you have apparently no problem with the `opt` above...